test/test_tree.rb

```diff
@@ -956,8 +954,7 @@ def test_content
     # instead of depth.  This method has been deprecated in this release and may be removed in the future.
     def test_depth
       begin
-        require 'structured_warnings'
-        assert_warn(DeprecatedMethodWarning) { do_deprecated_depth }
+        assert_not_empty(capture_output { do_deprecated_depth }.last)
       rescue LoadError
```

Comment:
rescuing LoadError doesn't seem to be needed because `require 'structured_warnings'` was removed.